### PR TITLE
Improve dynamic fee e2e test - Closes #3150

### LINKED ISSUE #3150: Add e2e custom fee validation (transfer&register delegate)
### Description

Send LSK and register delegate should test that it is not posible to place transaction with fees that are out of bounds.
It also should test that selecting low, medium and high priority changes fee value.

### Motivation

Currently we are relying on unit test for dynamic fee testing. It will be better to cover it also with e2e testing.

### Acceptance Criteria

- Send e2e test checks that go to confirmation button is disabled when custom fee is less than the minimum
- Send e2e test checks that go to confirmation button is disabled when custom fee is more than the hard cap value
- Register delegate e2e test checks that go to confirmation button is disabled when custom fee is less than the minimum
- Register delegate e2e test checks that go to confirmation button is disabled when custom fee is more than the hard cap value
- Selecting low, medium and high priority changes fee value

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?
This PR resolves #3150

### How was it solved?
- Add e2e steps

### How was it tested?
e2e test

### CODE REVIEW

File: src/utils/api/lsk/account.js [hunk 1]
```diff
@@ -29,7 +29,11 @@ export const getAccount = params =>
         ? { serverPublicKey: res.data[0].publicKey, ...res.data[0] }
         : { balance: 0 };
 
-      resolve({ ...offlineInfo, ...onlineInfo });
+      resolve({
+        ...offlineInfo,
+        ...onlineInfo,
+        publicKey: offlineInfo.publicKey || onlineInfo.publicKey,
```
Comment (UsamaHameed) [inline src/utils/api/lsk/account.js:35]: Can't wet just do this instead? 
```
resolve({
    ...onlineInfo,
    ...offlineInfo,
})
```

File: test/constants/selectors.js [hunk 2]
```diff
@@ -246,6 +246,10 @@ const ss = {
   addBookmarkIcon: '.add-bookmark-icon',
   inputLabel: '.input-label',
   saveButton: '.save-button',
+  lowPriorityFee: '.option-Low',
```
Comment (UsamaHameed) [inline test/constants/selectors.js:249]: we have classnames for all three options just a few lines above ^^

File: test/cypress/features/common/common.js [hunk 3]
```diff
@@ -213,3 +213,25 @@ And(/^I search for account ([^s]+)$/, function (string) {
...
+    generateFeePromise(ss.mediumPriorityFee),
+    generateFeePromise(ss.highPriorityFee),
+  ]).then((values) => {
+      const lowFee = values[0];
+      const mediumFee = values[1];
+      const highFee = values[2];
+      cy.wrap(null).should(() => {
+        expect(lowFee).to.not.equal(mediumFee);
+        expect(lowFee).to.not.equal(highFee);
+        expect(mediumFee).to.not.equal(highFee);
```
Comment (UsamaHameed) [inline test/cypress/features/common/common.js:234]: `expect(mediumFee).not.toEqual(...)` and so on

**Review by UsamaHameed — APPROVED** [review]

**Review by UsamaHameed — CHANGES_REQUESTED** [review]
Approve changes by mistake

File: test/cypress/features/common/common.js (hunk 3 above)
Comment (isalga) [reply test/cypress/features/common/common.js:234]: It is using chai's expect that is bundled in cypress, so `toEqual` does not exist.

**Review by UsamaHameed — APPROVED** [review]

**Review by reyraa — APPROVED** [review]

**Pull request merged**